client/components/AddSponsor.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+interface AddSponsorProps {
+  eventId: string;
+  chapterId: string;
+}
+
+interface SponsorData {
+  name: string;
+  website: string;
+  type: 'FOOD' | 'BEVERAGE' | 'OTHER'; // TODO: Add VENUE
+}
+
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+  max-width: 25%;
+`;
+
+const Input = styled.input`
+  margin-bottom: 15px;
+`;
+
+const ResponseDiv = styled.div`
+  margin: 15px 0;
+`;
+// TODO: Style div based on response reveived
+
+const AddSponsor: React.FC<AddSponsorProps> = ({ eventId, chapterId }) => {
```

Comment:
regarding this i think you can just do `const AddSponsor = ({ eventId, chapterId }: AddSponsorProps) => {` but someone else might have opinions on whether using `React.FC` is better

Reply:
I think `React.FC` is better as it automatically adds children for you, and check the return type

Reply:
I have added a return type of JSX.Element to AddSponsor but still getting eventId and chapterId are not validated. I need to guidance on how to fix these please?

Reply:
I am flying, but will help you out in evening 😅

Reply:
Oh I just realized this was an eslint error, not a typescript error.  We should turn off the `react/prop-types` rule since we're not using `PropTypes`.  Can you add this to `eslintrc.json`?  Just a new line under `rules` that says something like `"react/prop-types": 0`.

Once done, you can also get rid of the return type you added, since it is implied by the JSX you're returning in the FC (if you're always returning the same thing TS will be able to infer the type)